stdlib/public/core/FloatingPointTypes.swift.gyb

```diff
@@ -76,13 +86,19 @@ public struct ${Self} {
   }
 }
 
+${Availability(bits)}
 extension ${Self}: CustomStringConvertible {
   /// A textual representation of the value.
   public var description: String {
     if isNaN {
       return "nan"
     } else {
+% if bits == 16:
+      // TODO: replace with real _float16ToString implementation
+      var (buffer, length) = _float32ToString(Float(self), debug: false)
```

Comment:
Rather than having three copies of this conditional here, feel free to add a new stub for this to stdlib/public/stubs/Stubs.cpp and assign a Radar to me to fill in the SwiftDtoa function.
```
SWIFT_CC(swift) SWIFT_RUNTIME_STDLIB_API
uint64_t swift_float16ToString(char *Buffer, size_t BufferLength,
                               float Value, bool Debug) {
  // FIXME: dispatch to float for now, fill in float16 soon
  return swift_format_float(Value, Buffer, BufferLength);
}
```

Reply:
Yeah, this turns out to be quite a bit nicer. I'll do it this way.